task/06-conditions-n-loops-tasks.js

```diff
@@ -63,7 +77,11 @@ function getFactorial(n) {
  *   -1,1  =>  0  ( = -1 + 0 + 1 )
  */
 function getSumBetweenNumbers(n1, n2) {
-    throw new Error('Not implemented');
+    let result = 0;
+    while (n2 >= n1)
```

Comment:
it's not a good practice to change arguments if its not required. Also it's better to use for-loop when you know the number of step in advance